// ngx-slick-carousel/esm5/index.js
/**
 * @fileoverview added by tsickle
 * Generated from: index.ts
 * @suppress {checkTypes,constantProperty,extraRequire,missingOverride,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { CommonModule } from '@angular/common';
import { NgModule } from '@angular/core';
import { SlickCarouselComponent, SlickItemDirective } from './slick.component';
import * as ɵngcc0 from '@angular/core';
export { SlickCarouselComponent, SlickItemDirective } from './slick.component';
var SlickCarouselModule = /** @class */ (function () {
    function SlickCarouselModule() {
    }
SlickCarouselModule.ɵmod = ɵngcc0.ɵɵdefineNgModule({ type: SlickCarouselModule });
SlickCarouselModule.ɵinj = ɵngcc0.ɵɵdefineInjector({ factory: function SlickCarouselModule_Factory(t) { return new (t || SlickCarouselModule)(); }, imports: [[
            CommonModule
        ]] });
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(SlickCarouselModule, { declarations: function () { return [SlickCarouselComponent,
        SlickItemDirective]; }, imports: function () { return [CommonModule]; }, exports: function () { return [SlickCarouselComponent,
        SlickItemDirective]; } }); })();
/*@__PURE__*/ (function () { ɵngcc0.ɵsetClassMetadata(SlickCarouselModule, [{
        type: NgModule,
        args: [{
                imports: [
                    CommonModule
                ],
                declarations: [
                    SlickCarouselComponent,
                    SlickItemDirective,
                ],
                exports: [
                    SlickCarouselComponent,
                    SlickItemDirective,
                ]
            }]
    }], function () { return []; }, null); })();
    return SlickCarouselModule;
}());
export { SlickCarouselModule };
